modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/ApplicationManagementUserRegistrantSuccessTest.java

```diff
@@ -0,0 +1,170 @@
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.wso2.identity.integration.test.rest.api.server.application.management.v1.Utils.assertNotBlank;
+import static org.wso2.identity.integration.test.rest.api.server.application.management.v1.Utils.extractApplicationIdFromLocationHeader;
+
+import java.io.IOException;
+
+/**
+ * Test class for Application Management User Registrant Success Test.
+ */
+public class ApplicationManagementUserRegistrantSuccessTest extends ApplicationManagementBaseTest {
+
+    private String createdAppId;
+    static final String APPLICATION_USER_REGISTRANT_CONTEXT = "/user-registrants";
+
+    @Factory(dataProvider = "restAPIUserConfigProvider")
+    public ApplicationManagementUserRegistrantSuccessTest(TestUserMode userMode) throws Exception {
+
+        super(userMode);
+    }
+
+    @BeforeTest(alwaysRun = true)
+    public void initTestClass() throws IOException {
+
+        super.init();
+    }
+
+    @AfterTest(alwaysRun = true)
+    public void testFinish() {
+
+        super.testFinish();
+    }
+
+    @Test
+    public void testRetrieveUserRegistrantsOfInvalidApp() {
+
+        Response response = getResponseOfGet(APPLICATION_MANAGEMENT_API_BASE_PATH + PATH_SEPARATOR + "some-wrong-id" +
+                APPLICATION_USER_REGISTRANT_CONTEXT);
+
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("totalResults", equalTo(0))
+                .body("userRegistrants.size()", equalTo(0));
+    }
+
+    @Test
+    public void testCreateAppWithNoAuthenticationOptions() throws Exception {
+
+        String body = readResource("create-app-with-no-authenticators.json");
+
+        Response response = getResponseOfPost(APPLICATION_MANAGEMENT_API_BASE_PATH, body);
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .header(HttpHeaders.LOCATION, notNullValue());
+
+        String location = response.getHeader(HttpHeaders.LOCATION);
+        createdAppId = extractApplicationIdFromLocationHeader(location);
+        assertNotBlank(createdAppId);
+    }
+
+    @Test(dependsOnMethods = "testCreateAppWithNoAuthenticationOptions")
+    public void testRetrieveUserRegistrantsDefault() {
+
+        Response response = getResponseOfGet(APPLICATION_MANAGEMENT_API_BASE_PATH + PATH_SEPARATOR + createdAppId +
+                APPLICATION_USER_REGISTRANT_CONTEXT);
+
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("totalResults", equalTo(1))
```

Comment:
Wondering whether we should check the default is properly coming. We currently have a fixed default so checking that should be okay.

Reply:
Did you mean that always the default will be the Basic Authenticator?

Reply:
Yes always basic authenticator will be the default according to current configurations.